Dockerfile

```diff
@@ -0,0 +1,9 @@
+# ----- Base -----
+FROM node:16
+WORKDIR /usr/app
+
+# ----- Install & Build -----
+COPY ["package.json", "package-lock.json", "babel.config.json", "./"]
```

Comment:
Consider looking into a potential redundancy between "./" and line 8 ".".
Looks like you're copying the entire root on line 6 making line 8 redundant.
At the same time, lines 6 and 7 are all about getting dependencies insider your image, so it might be better to avoid copying the entire root on line 6.